torch/_functorch/functional_call.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
nit: don't think we need the header anymore. It's not used for pytorch/pytorch but is required for all the other repos (pytorch/functorch, etc)